packages/plugins/src/ERC6900/ERC6900ZkEmailRecoveryModule.sol

```diff
@@ -0,0 +1,365 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import {BasePlugin} from "erc6900-reference-implementation/plugins/BasePlugin.sol";
+import {IPluginExecutor} from "erc6900-reference-implementation/interfaces/IPluginExecutor.sol";
+import {ManifestFunction, ManifestAssociatedFunctionType, ManifestAssociatedFunction, PluginManifest, PluginMetadata, IPlugin} from "erc6900-reference-implementation/interfaces/IPlugin.sol";
+import {MockGroth16Verifier} from "../safe/utils/MockGroth16Verifier.sol";
+import {MockDKIMRegsitry} from "../safe/utils/MockDKIMRegsitry.sol";
+import {IDKIMRegsitry} from "../safe/interface/IDKIMRegsitry.sol";
+
+/*//////////////////////////////////////////////////////////////////////////
+    THIS CONTRACT IS STILL IN ACTIVE DEVELOPMENT. NOT FOR PRODUCTION USE        
+//////////////////////////////////////////////////////////////////////////*/
```

Comment:
Thanks for adding these. In the future, I think we may need a better way to delineate this, such as placing them in a different directory when ready for production use.